third.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func Average(inputArray []int) float64 {
+	return float64(CalculateSumOfArray(inputArray)) / float64(len(inputArray))
```

Comment:
What happens if len(inputArray) == 0 ?
Please add a check that slice length is greater than 0.